src/app/routes/utils/getRadioService/index.js

```diff
@@ -6,8 +6,14 @@ const RADIO_SERVICE_MAPPINGS = {
 };
 
 export default ({ service, pathname }) => {
-  if (service === 'persian' && pathname.includes('bbc_persian_radio')) {
+  const isPodcast = pathname.includes('podcasts');
+
+  if (
+    service === 'persian' &&
+    (pathname.includes('bbc_persian_radio') || isPodcast)
+  ) {
```

Comment:
```suggestion
  const isPodcast = pathname.includes('podcasts');
  const isPersianRadio = service === 'persian' && pathname.includes('bbc_persian_radio')

  if (isPersianRadio || isPodcast) {
```

Reply:
Just spotted something after e2es were failing - the condition required is slightly different to the suggestion. What we need is:

1. If the service is 'persian' 
2. AND the pathname includes 'bbc_persian_radio' OR 'podcasts'
3. then return 'persian'

This code suggestion will return `persian` for any schedule request that includes 'podcasts'.

Would this be an improvement on the initial code?

```js
  const isPodcast = pathname.includes('podcasts');
  const isPersianRadioMasterBrand = pathname.includes('bbc_persian_radio');

  if (service === 'persian' && (isPersianRadioMasterBrand || isPodcast)) {
    return 'persian';
  }
```